pkg/cnb/env_vars.go

```diff
@@ -27,3 +28,37 @@ type envVariable struct {
 	Name  string `json:"name" toml:"name"`
 	Value string `json:"value" toml:"value"`
 }
+
+type buildEnvVariable struct {
+	Env []envVariable `toml:"env"`
+}
+
+type envBuildVariable struct {
+	Env []envVariable
+}
+
+func (a *envBuildVariable) UnmarshalTOML(f interface{}) error {
```

Comment:
this will prioritize `io.buildpacks.build` over legacy pack's `io.buildpacks.env.build` but I think it's ok

Reply:
In fact, I think it is not giving any priority, it is just parsing both cases depending on what the user specifies because they go to different **case** in the switch.

Reply:
yea I think this is ok b/c `io.buildpacks.env` and `io.buildpacks.env.build` is not valid toml